src/test_l3_dsDisplay.c

```diff
@@ -0,0 +1,433 @@
+
+    DS_ASSERT(status == dsERR_NONE);
+
+    UT_LOG_MENU_INFO("Calling dsRegisterDisplayEventCallback");
+
+    UT_LOG_INFO("Calling dsGetEDID(IN:handle:[0x%0X], IN:cb[0x%0X]) ", gDisplayHandle, displayEventCallback);
+
+    status = dsRegisterDisplayEventCallback(gDisplayHandle,displayEventCallback);
+
+    UT_LOG_INFO("Result dsRegisterDisplayEventCallback(IN:handle:[0x%0X], IN:cb[0x%0X]), dsError_t=[%s]", gDisplayHandle, displayEventCallback, UT_Control_GetMapString(errorMappingTable, status));
+
+    DS_ASSERT(status == dsERR_NONE);
+
+
+exit:
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+
+}
+
+/**
+ * @brief Retrieves Extended Display Identification Data (EDID) for the display.
+ *
+ * Retrieves the EDID information for the display associated with the selected handle.
+ *
+ * **Test Group ID:** 03@n
+ * **Test Case ID:** 003@n
+ *
+ * **Test Procedure:**
+ * Refer to Test specification documentation
+ * [ds-display_L3_Low-Level_TestSpecification.md](../../docs/pages/ds-display_L3_Low-Level_TestSpecification.md)
+ */
+void test_l3_dsDisplay_get_edid(void) 
+{
+    gTestID = 3;
+    UT_LOG_INFO("In %s [%02d%03d]", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status = dsERR_NONE;
+
+    if (gDisplayHandle == -1) {
+        UT_LOG_ERROR("Failed to get display handle\n");
+        goto exit;
+    }
+
+    UT_LOG_INFO("Calling dsGetEDID(IN:handle:[0x%0X], OUT:edid:[])", gDisplayHandle);
+    status = dsGetEDID(gDisplayHandle, &edid);
+
+    UT_LOG_INFO("Result dsGetEDID(dsDisplayEDID_t(productCode:[%d], serialNumber:[%d], manufactureYear:[%d], manufactureWeek:[%d], hdmiDeviceType:[%s], isRepeater:[%s], physicalAddressA:[%u], physicalAddressB:[%u], physicalAddressC:[%u], physicalAddressD:[%u], numOfSupportedResolution:[%d], monitorName:[%s])", \
+    edid.productCode, edid.serialNumber, edid.manufactureYear, edid.manufactureWeek, edid.hdmiDeviceType, edid.isRepeater, edid.physicalAddressA, edid.physicalAddressB, edid.physicalAddressC, edid.physicalAddressD, edid.numOfSupportedResolution, edid.monitorName);
+
+    DS_ASSERT(status == dsERR_NONE);
+
+exit:
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+ * @brief Retrieves the Extended Display Identification Data (EDID) bytes for the display.
+ *
+ * Retrieves the EDID data bytes for the display associated with the selected handle.
+ *
+ * **Test Group ID:** 03@n
+ * **Test Case ID:** 004@n
+ *
+ * **Test Procedure:**
+ * Refer to Test specification documentation
+ * [ds-display_L3_Low-Level_TestSpecification.md](../../docs/pages/ds-display_L3_Low-Level_TestSpecification.md)
+ */
+void test_l3_dsDisplay_get_edidbytes(void)
+{
+    gTestID = 4;
+    UT_LOG_INFO("In %s [%02d%03d]", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status = dsERR_NONE;
+    int length = 0;
```

Comment:
use the standard data types (int32_t), check other places

Reply:
Done